plutus-ledger/src/Ledger/Validation.hs

```diff
@@ -114,7 +114,7 @@ addressed by extending the emulator, without having to bring in the full block
 validating machinery.
 
 * We cannot represent different eras - everything is 'AlonzoEra'.
-* There is no handling of epoch boundaries, rewards, etc.
+* There is no handling of rewards, etc.
```

Comment:
epoch boundaries is something else, we still don't handle those.

Reply:
my bad, I'm reverting the initial comment.